aten/src/ATen/native/cpu/UnaryOpsKernel.cpp

```diff
@@ -712,6 +712,19 @@ static void modified_bessel_k1_kernel(TensorIteratorBase& iterator) {
   }                                                                                              \
   REGISTER_DISPATCH(op##_stub, &CPU_CAPABILITY::op##_kernel)
 
+#define STATIC_IMPLEMENT_COMPLEX_KERNEL(op)                                                      \
```

Comment:
I guess you cannot update the current IMPLEMENT_COMPLEX_KERNEL to use static like you did for the non-complex version?

Reply:
Yes, some instantiations are not static